.github/workflows/swift-toolchain.yml

```diff
@@ -2317,6 +2327,12 @@ jobs:
           Echo CERTIFICATE=$PFXPath | Out-File -FilePath ${env:GITHUB_ENV} -Encoding utf8 -Append
         if: ${{ needs.context.outputs.signed }}
 
+      # TODO(issues/154): Preload WixToolset.Sdk on Azure images so we can remove this step.
+      - name: Install WixToolset.Sdk
```

Comment:
We could also do this on upstream conditionally if it's not installed

Reply:
I think we should commit it here, but I don't feel strongly about it: It's a short term change that's specific to us since we're using different VMs. Once we install `WixToolset.Sdk` on the Azure images we should just delete it.